src/web/mjs/engine/Settings.mjs

```diff
@@ -232,7 +254,8 @@ export default class Settings {
             } );
             document.dispatchEvent( new CustomEvent( EventListener.onSettingsChanged, { detail: this } ) );
         } catch( error ) {
-            console.error('Failed to load HakuNeko settings!', error);
+            console.error('Failed to load HakuNeko settings! New settings file created.', error);
+            this.saveProfile();
```

Comment:
This should be done for the expected condition (`File does not Exist`) somewhere in the try block.
This catch block catches any exception, including errors like `Out of Memory`, `Permission Denied`, ...
calling `saveProfile` may lead to further re-throw errors.

Reply:
ahh alright, i figured it was for loading the settings only